client/src/modules/list/components/Items/ItemsList.jsx

```diff
@@ -81,7 +115,7 @@ class ItemsList extends PureComponent {
     return archived ? (
       <ul className="items-list">
         <TransitionGroup component={null}>
-          {items.slice(0, limit).map(item => (
+          {displayedItems.map(item => (
```

Comment:
Please revert to purely based slicing if possible.

Reply:
Not possible with the current implementation. 
I've created a task to reimplement it, to pass already sliced items from parent component.